### PR TITLE
Cleanup codebase

### PR DESCRIPTION
**Pull request by vytskalt**
I cleaned up the project to hopefully make it easier to work with and also fixed a couple of bugs while doing it.
Keep up the great work!

### CODE REVIEW

File: TimoCloud-API/src/main/java/cloud/timo/TimoCloud/api/messages/objects/PluginMessage.java [hunk 1]
```diff
@@ -8,7 +8,7 @@
  * A plugin message is an object just containing data.
  * It basically consists of a map where data can be stored.
  * The class provides methods to retrieve certain data types easily.
- *
+ * <p>
```
Comment (TimoZikeli) [inline TimoCloud-API/src/main/java/cloud/timo/TimoCloud/api/messages/objects/PluginMessage.java]: Why did you add (just a) paragraph starting tag?

File: TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/bukkit/managers/SignManager.java [hunk 2]
```diff
@@ -75,15 +83,17 @@ private void loadSignTemplates() {
             List<String> sortOutStates = config.contains(sortOutStatesKey) ? config.getStringList(template + ".sortOutStates") : null;
             try {
                 for (String layout : config.getConfigurationSection(template + ".layouts").getKeys(false)) {
-                    List<String>[] lines = Arrays.copyOf(new Object[4], 4, List[].class);
+                    List[] lines = Arrays.copyOf(new Object[4], 4, List[].class);
```
Comment (TimoZikeli) [inline TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/bukkit/managers/SignManager.java]: Why did you remove the type information?

File: TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/core/objects/Base.java [hunk 3]
```diff
@@ -138,19 +153,17 @@ public void onDisconnect() {
...
-                int availableRam = Math.max(0, ((Number) map.get("freeRam")).intValue() - getKeepFreeRam());
-                setAvailableRam(Math.max(0, Math.min(availableRam, maxRam - usedRam)));
-                double cpuLoad = (Double) map.get("cpuLoad");
-                setCpuLoad(cpuLoad);
-                boolean ready = (boolean) map.get("ready") && cpuLoad <= getMaxCpuLoad();
-                setReady(ready);
-                break;
-            default:
-                sendMessage(message);
+        if (type == MessageType.BASE_RESOURCES) {
```
Comment (TimoZikeli) [inline TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/core/objects/Base.java]: Please leave this as it was since there could be more messages added.

**Review by TimoZikeli — CHANGES_REQUESTED** [review]
Thanks a lot for the reformatting! 
I only noticed 3 issues.

**Comment by TimoZikeli** [comment]
(+ please bump the version number)

File: TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/core/objects/Base.java (hunk 3 above)
Comment (VytskaLT) [reply TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/core/objects/Base.java:156]: That's Intellij's auto reformatting feature

**Comment by vytskalt** [comment]
All of those issues were Intellij's auto reformatting feature, sorry about that

**Comment by vytskalt** [comment]
fixed :)

**Comment by TheMagBum** [comment]
Could you possibly resolve the merge conflicts?

**Comment by vytskalt** [comment]
> Could you possibly resolve the merge conflicts?

no sorry I don't have the time right now

**Comment by TheMagBum** [comment]
Then please still do that when you find the time

**Comment by vytskalt** [comment]
I resolved the conflicts, but please check it out because I might have messed up something
